src/axom/inlet/SphinxWriter.cpp

```diff
@@ -213,6 +269,130 @@ void SphinxWriter::writeAllTables()
   }
 }
 
+void SphinxWriter::writeNestedTables()
+{
+  // Used to avoid duplicate hyperlinks
+  int containerCount = 0;

```

Comment:
(This references a comment I made on the Conversation page)
It looks like the table indices are `inlet`-generated.

My concern is that these indices are meaningful to the system (e.g. to generate unique links), but not to the user.
Is there a way to use these numbers but not display them?

Reply:
It looks like Axom is enabling this explicitly:

https://github.com/LLNL/axom/blob/858899f671ec6d596f48e2102a634e32215d6e4a/src/conf.py#L91-L93

See https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-numfig for a description of the option - should we just turn this off then (the default)?

Reply:
Thanks @joshessman-llnl  -- I think so.

We might need to reword a few things in the mint docs, which, I think might be referencing these numbers.
(which seems a bit formal for sphinx docs to me).

Reply:
I've gone ahead and tried this out - see commit https://github.com/LLNL/axom/pull/508/commits/8834021be87f50e0e9e67fb6ddce2eb9fc682b1c

Should this maybe be moved into a separate PR?